apps/emqx_dashboard/src/emqx_dashboard_swagger.erl

```diff
@@ -457,7 +457,18 @@ trans_description(Spec, Hocon) ->
             Spec;
         Desc ->
             Desc1 = binary:replace(Desc, [<<"\n">>], <<"<br/>">>, [global]),
-            Spec#{description => Desc1}
+            maybe_add_summary_from_label(Spec#{description => Desc1}, Hocon)
+    end.
+
+maybe_add_summary_from_label(Spec, Hocon) ->
```

Comment:
these two clauses can reduce to one

Reply:
It's not clear to me which clauses you are referring to, can you elaborate please?